### ISSUE TITLE
Input tags - Allow removing Tags with the Keyboard

### ISSUE THREAD

**Issue by turolopezsanabria** [SUI-Components/SUIComponents#1751]
Component [Input tags](https://sui-components.vercel.app/workbench/molecule/inputTags/demo)

**Is your iteration request related to a problem?**
At this moment, it's imposible to delete tags with the keyboard

**Describe the solution you'd like**
Accessibility: It should be able to focus, navigate and delete `tags` using the Keyboard

**Comment by Xaconi** [comment]
Hey @turolopezsanabria can I work on this one?

EDIT: started already installing the local environment and after some troubles (I really missed a local environment development guide), I ended here when I use `npx sui-studio dev @s-ui/react-molecule-input-tags`:

![image](https://user-images.githubusercontent.com/1944438/137212420-b1551663-81dc-4f1c-b003-4330f2172104.png)

What I'm doing wrong? How can I develop on a local environment? Is related with babel presets?

Regards!

**Comment by andresin87** [comment]
take a look on https://dev.to/adevintaspain/adevinta-s-hacktoberfest-2021-s-ui-components-project-1bg9 @stefi23 post to get ready and start coding (@Xaconi). Enjoy!

**Comment by andresin87** [comment]
error fixed.

Instal dependencies again using:
https://github.com/SUI-Components/sui-components/issues/1744#issuecomment-945758050

in your case
`npm run dev molecule/inputTags`

**Comment by Xaconi** [comment]
Found another error on `npm run phoenix`

![image](https://user-images.githubusercontent.com/1944438/138157226-ce8f9d71-6ad7-4db4-a4dd-98ffb77fbce9.png)

**Comment by Xaconi** [comment]
Hey @andresin87 tried with `npx @s-ui/mono@2 run 'rm -rf ./node_modules' && rm -rf ./node_modules && npm install` instead `npm run phoenix` and the command executed OK, but the next one (`npm run dev molecule/dropdownList` or `sui-studio dev molecule/dropdownList` fails with this error code:

![image](https://user-images.githubusercontent.com/1944438/138611476-1a2a3245-431d-4bd3-932a-71fa8113bb08.png)

**Comment by andresin87** [comment]
```
$ rm -rf ./node_modules && rm -f package-lock.json && npm install --prefer-online --legacy-peer-deps
$ npm run phoenix
$ npm run dev molecule/inputTags
```

**Comment by Xaconi** [comment]
@andresin87 @turolopezsanabria 

`npm run phoenix`

![image](https://user-images.githubusercontent.com/1944438/138773135-fd0ecb21-a42f-4f4c-bb35-44e2ab949e61.png)

`npm run dev molecule/inputTags`

![image](https://user-images.githubusercontent.com/1944438/138773206-38989ca8-b542-4d9b-9b8c-1eea95c72d40.png)

😭😭😭

**Comment by andresin87** [comment]
Windows or Mac?

**Comment by Xaconi** [comment]
Windows 10

**Comment by Xaconi** [comment]
@andresin87 @turolopezsanabria Any updates on this one??
